src/bin/mdev.rs

```diff
@@ -13,6 +13,7 @@ use nix::{
     sys::stat::{makedev, mknod, Mode, SFlag},
     unistd::{chown, unlink},
 };
+use syslog_tracing::Syslog;
```

Comment:
If someone uses `default-features = false` this code won't build anymore?

Reply:
true